src/DayOfWeek.php

```diff
@@ -184,64 +182,65 @@ public function is(int $dayOfWeek) : bool
      * do *not* use strict object comparison to compare two DayOfWeek instances,
      * as it is possible to get a different instance for the same day using serialization.
      */
-    public function isEqualTo(DayOfWeek $that) : bool
+    public function isEqualTo(DayOfWeek $that): bool
     {
         return $this->value === $that->value;
     }
 
     /**
      * Returns whether this DayOfWeek is Monday to Friday.
      */
-    public function isWeekday() : bool
+    public function isWeekday(): bool
     {
         return $this->value >= self::MONDAY && $this->value <= self::FRIDAY;
     }
 
     /**
      * Returns whether this DayOfWeek is Saturday or Sunday.
      */
-    public function isWeekend() : bool
+    public function isWeekend(): bool
     {
         return $this->value === self::SATURDAY || $this->value === self::SUNDAY;
     }
 
     /**
      * Returns the DayOfWeek that is the specified number of days after this one.
      */
-    public function plus(int $days) : DayOfWeek
+    public function plus(int $days): DayOfWeek
     {
         return DayOfWeek::get((((($this->value - 1 + $days) % 7) + 7) % 7) + 1);
     }
 
     /**
      * Returns the DayOfWeek that is the specified number of days before this one.
      */
-    public function minus(int $days) : DayOfWeek
+    public function minus(int $days): DayOfWeek
     {
-        return $this->plus(- $days);
+        return $this->plus(-$days);
```

Comment:
Doubt on this one. Does that mean that we write `-$b`, but `$a - $b`?

Reply:
Yes, here are the two rules in play here:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/doc/rules/operator/unary_operator_spaces.rst
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/doc/rules/operator/binary_operator_spaces.rst